src/payment/strategies/stripev3/stripev3-payment-strategy.ts

```diff
@@ -79,6 +85,20 @@ export default class StripeV3PaymentStrategy implements PaymentStrategy {
             // tslint:disable-next-line: variable-name
             const { instrumentId: token, ccNumber: credit_card_number_confirmation, ccCvv: verification_value } = paymentData;
 
+            if (this._isHostedPaymentFormEnabled(methodId, gatewayId) && this._shouldRenderHostedForm) {
+                const form = this._hostedForm;
+
+                if (!form) {
+                    throw new NotInitializedError(NotInitializedErrorType.PaymentNotInitialized);
+                }
+
+                try {
+                    await form.validate();
+                } catch (error) {
+                    throw new Error(error.message);
```

Comment:
why does this need to be caught and rethrown? This is wrapping a specific `InvalidHostedFormFields` error for a generic Error type. This might end up showing an error modal in checkout-js, rather than an inline error.

Reply:
Perhaps we could use a conditional catch block then?